pkg/migrations/op_add_column.go

```diff
@@ -268,6 +285,11 @@ func (o *OpAddColumn) addCheckConstraint(ctx context.Context, tableName string,
 	return err
 }
 
+// UniqueIndexName returns the name of the unique index for the given column
+func UniqueIndexName(columnName string) string {
+	return columnName + "uniq"
+}
```

Comment:
Elsewhere, these temporary names use `_pgroll` prefixes. We should follow that patten here too:

```suggestion
func UniqueIndexName(columnName string) string {
	return "_pgroll_uniq_"  + columnName + "uniq"
}
```